src/sentry/options/defaults.py

```diff
@@ -45,6 +45,11 @@
     default=0,
     flags=FLAG_ALLOW_EMPTY | FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
 )
+register(
+    "system.debug-files-renewal-age-threshold",
```

Comment:
```suggestion
    "system.debug-files-renewal-age-threshold-days",
```

Maybe call out its days explicitly